<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>335</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_1000">
           <property name="text">
            <string>$1000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_500">
           <property name="text">
            <string>$500</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_200">
           <property name="text">
            <string>$200</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_100">
           <property name="text">
            <string>$100</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_50">
           <property name="text">
            <string>$50</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>$20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>$10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>$5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>$3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>$1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QSpinBox" name="spinBox_1000">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_500">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_200">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_100">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_50">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_20">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_10">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_5">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_3">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_1">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_st1000">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st500">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st200">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st100">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st50">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st20">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st10">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st5">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_st1">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_total">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 255);</string>
         </property>
         <property name="text">
          <string>Total:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_moneySymbol">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>$</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_totalCount">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
